plutus-core/cost-model/data/builtinCostModel.json

```diff
@@ -691,5 +691,37 @@
             "arguments": 10,
             "type": "constant_cost"
         }
+    },
+    "showInteger": {
+        "cpu": {
+            "arguments": {
+                "intercept": 0,
+                "slope": 0
+            },
+            "type": "linear_cost"
+        },
+        "memory": {
+            "arguments": {
+                "intercept": 0,
+                "slope": 0
+            },
+            "type": "linear_cost"
+        }
+    },
+    "showByteString": {
+        "cpu": {
+            "arguments": {
+                "intercept": 0,
+                "slope": 0
+            },
+            "type": "linear_cost"
+        },
+        "memory": {
+            "arguments": {
+                "intercept": 0,
```

Comment:
I guess these should have the numbers produced by `generate-cost-model`.  I suppose we'll have new figures for every builtin, so I wonder if we should update the entire cost model to make it consistent with the current evaluator.  That might need us to update the machine step costs as well.  I suppose we could compare the new JSON output with the old version to see if things have changed much since the last update.

Reply:
I we do update the cost model then we should check in `benching.csv` too, just so we have a record of the data.  I found that not having it checked in made it hard to find the data if I wanted to investigate the cost model.